### PR TITLE
more storybook

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: packages/common/src/components/GithubBadge/index.tsx [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+
+const DivOrA = ({ href, ...props }: DivOrAProps) =>
   href ? (
     <StyledA target="_blank" rel="noopener noreferrer" href={href} {...props} />
   ) : (
     <div {...props} />
   );
 
-function GithubBadge({ username, repo, url, branch, ...props }) {
+function GithubBadge({ username, repo, url, branch, ...props }: BadgeProps) {
```
Comment (MichaelDeBoey) [inline packages/common/src/components/GithubBadge/index.tsx]:
```suggestion
const GithubBadge: FunctionComponent<BadgeProps> = ({ username, repo, url, branch, ...props }) => {
```

File: packages/common/src/components/GithubBadge/index.tsx [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+  url?: string;
+  branch: string;
+};
+
+type DivOrAProps = {
+  href?: string;
+  children: React.ReactNode;
+};
+
+const DivOrA = ({ href, ...props }: DivOrAProps) =>
```
Comment (MichaelDeBoey) [inline packages/common/src/components/GithubBadge/index.tsx]:
```suggestion
const DivOrA: FunctionComponent<DivOrAProps> = ({ href, ...props }) =>
```

File: packages/common/src/components/GithubBadge/index.tsx [hunk 3]
```diff
@@ -1,15 +1,27 @@
-import * as React from 'react';
+import React from 'react';
```
Comment (MichaelDeBoey) [inline packages/common/src/components/GithubBadge/index.tsx]:
```suggestion
import React, { FunctionComponent } from 'react';
```

File: packages/common/src/components/Input/index.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import { storiesOf } from '@storybook/react';
+import Input from './';
+import { ThemeDecorator } from '../../stories/decorators';
+
+const stories = storiesOf('components/Input', module).addDecorator(
+  ThemeDecorator
+);
+
+stories.add('Basic Input', () => <Input />);
+stories.add('Block Input', () => <Input />);
```
Comment (MichaelDeBoey) [inline packages/common/src/components/Input/index.stories.tsx]: You can just chain all `add` calls

```suggestion
    .add('Block Input', () => <Input />);
```

Comment (SaraVieira) [reply packages/common/src/components/Input/index.stories.tsx]: Done in all! Thank you!

File: packages/common/src/components/Input/index.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+import Input from './';
+import { ThemeDecorator } from '../../stories/decorators';
+
+const stories = storiesOf('components/Input', module).addDecorator(
+  ThemeDecorator
+);
+
+stories
+  .add('Basic Input', () => <Input />)
+  .add('Block Input', () => <Input />)
```
Comment (shiftyp) [inline packages/common/src/components/Input/index.stories.tsx]: Are these two stories supposed to be identical?

Comment (SaraVieira) [reply packages/common/src/components/Input/index.stories.tsx]: Nooo, fixing it

**Comment by SaraVieira** [comment]
@shiftyp Your flex stories look amazing!

**Pull request merged**